seatunnel-connectors-v2/connector-pulsar/src/main/java/org/apache/seatunnel/connectors/seatunnel/pulsar/source/enumerator/PulsarSplitEnumerator.java

```diff
@@ -107,10 +109,10 @@ public PulsarSplitEnumerator(SourceSplitEnumerator.Context<PulsarPartitionSplit>
                                  StopCursor stopCursor,
                                  String subscriptionName,
                                  Set<TopicPartition> assignedPartitions) {
-        if ((partitionDiscoverer instanceof TopicPatternDiscoverer)
+        if (partitionDiscoverer instanceof TopicPatternDiscoverer
             && partitionDiscoveryIntervalMs > 0
             && Boundedness.BOUNDED == stopCursor.getBoundedness()) {
-            throw new IllegalArgumentException("Bounded streams do not support dynamic partition discovery.");
+            throw new PulsarConnectorException(PulsarConnectorErrorCode.UNSUPPORTED_OPERATION, "Bounded streams do not support dynamic partition discovery.");
```

Comment:
please use `CommonErrorCode.UNSUPPORTED_OPERATION` instead of it.

Reply:
Ok, thank you for your comment